config/build-config.yaml

```diff
@@ -1,6 +1,10 @@
 recipes:
+{% if cudatoolkit == '10.2' %}
```

Comment:
cudnn 7.6.5 should be built only on ppc for cudatoolkit 10.2.

Reply:
It's probably best to keep the platforms in sync.